src/components/container-not-offers/container-not-offers.tsx

```diff
@@ -0,0 +1,21 @@
+import {CitiesCoordinatesKeys} from '../../const/city-points.ts';
+import {useAppSelector} from '../../hooks/use-app-selector.ts';
+
+function ContainerNotOffers() {
+  const selectedCityStore: CitiesCoordinatesKeys = useAppSelector((state) => state.selectedCity);
+
+  return (
+    <div className="cities__places-container cities__places-container--empty container">
+      <section className="cities__no-places">
+        <div className="cities__status-wrapper tabs__content">
+          <b className="cities__status">No places to stay available</b>
+          <p className="cities__status-description">We could not find any property available at the moment in {` ${selectedCityStore}`}
```

Comment:
Нам не нужно оборачивать в бектики и делать интерполяцию
```suggestion
          <p className="cities__status-description">We could not find any property available at the moment in {selectedCityStore}
```